Vagrantfile

```diff
@@ -0,0 +1,157 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
+# configures the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+Vagrant.configure("2") do |config|
+  # The most common configuration options are documented and commented below.
+  # For a complete reference, please see the online documentation at
+  # https://docs.vagrantup.com.
+
+  # Every Vagrant development environment requires a box. You can search for
+  # boxes at https://atlas.hashicorp.com/search.
+  config.vm.box = "ubuntu/xenial64"
+
+  # Create a forwarded port mapping which allows access to a specific port
+  # within the machine from a port on the host machine and only allow access
+  # via 127.0.0.1 to disable public access
+  config.vm.network "forwarded_port", guest: 7474, host: 7474, host_ip: "127.0.0.1"
+  config.vm.network "forwarded_port", guest: 7473, host: 7473, host_ip: "127.0.0.1"
+  config.vm.network "forwarded_port", guest: 7472, host: 7472, host_ip: "127.0.0.1"
```

Comment:
This config links a specific port on the VM to a port on the host. Because Neo4j will be running inside a VM, ordinarily visiting `localhost:7474` in your browser would produce nothing. But this config forwards the VM 7474 port to the host machine allowing you to visit `localhost:7474` in your browser to get the neo4j server app.

By default though, Neo4j rejects non-local connections. And the port forwarding is a non-local connection. To get around this, I update neo4j.conf below to accept non-local connections.